src/test/java/com/google/firebase/auth/TestTokenFactory.java

```diff
@@ -86,4 +112,11 @@ public String createToken(JsonWebSignature.Header header, JsonWebToken.Payload p
     payload.setSubject(UID);
     return payload;
   }
+
+  public JsonWebSignature.Header createHeaderForEmulator() {
+    JsonWebSignature.Header header = new JsonWebSignature.Header();
+    header.setAlgorithm("NONE");
```

Comment:
It should be "none" as per the JWS spec.

Reply:
Done.